stories/State.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+
+export interface IState<T> {
+  initialState?: T;
+  children: (data: T, setState: ((state: T) => void)) => React.ReactNode;
+}
+
+interface IStateState<T> {
```

Comment:
all my react components have an `IFoo` which is the props interface (which is what's exported and people use) and a `IFooState` which is the state interface used internally to make TypeScript happy. This component just has Foo = "State" so... lol

Reply:
Got it. I can live with this because this is on storybook, but in case this would be a component anywhere else in the app I would try to think to a better name.